modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// TBD
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first}.
+
+.Prerequisites
+
+* Your {rh-openstack} deployment has an existing IPv6 subnet.
```

Comment:
I'd mention here that this ipv6 subnet should be dhcpv6-stateful only. @MaysaMacedo what do you think?

Reply:
```suggestion
* Your {rh-openstack} deployment has an existing IPv6 subnet that is DHCPv6-stateful.
```